DATA/production/configurations/asyncReco/setenv_extra.sh

```diff
@@ -408,6 +408,10 @@ has_detector FT0 && export ARGS_EXTRA_PROCESS_o2_tpcits_match_workflow="$ARGS_EX
 export ARGS_EXTRA_PROCESS_o2_tof_matcher_workflow="$ARGS_EXTRA_PROCESS_o2_tof_matcher_workflow --output-type matching-info,calib-info --enable-dia --use-fit"
 export CONFIG_EXTRA_PROCESS_o2_tof_matcher_workflow+=";$ITSEXTRAERR;$TRACKTUNETPC;$VDRIFTPARAMOPTION;"
 
+if [[ $ALIEN_JDL_LPMPASSNAME == "cpass0" ]]; then
+   CONFIG_EXTRA_PROCESS_o2_tof_matcher_workflow+="MatchTOF.nsigmaTimeCut=6;"
```

Comment:
Can you also add a `;` before `MatchTOF....`?

Reply:
Is it needed?
It is there two lines above

Reply:
$VDRIFTPARAMOPTION;"

Reply:
I know, but I always add one when there are vars with `+=` in order to avoid messing up passing the arguments. So if you could do it...

Reply:
done

Reply:
Thanks!